bootstrap.py

```diff
@@ -73,7 +73,7 @@
 
 options, args = parser.parse_args()
 if options.version:
-    print("bootstrap.py version %s" % __version__)
+    print(("bootstrap.py version %s" % __version__))
```

Comment:
This file should be deleted, it does not work with more recent Python versions.

Reply:
sure.